### PR TITLE
feat : Added Product listing page

### PR DESCRIPTION
**Pull request by BalajiHarinath**
I have added the Product listing page for the E-commerce app. Please review.

### CODE REVIEW

File: Pages/cart page/cart.html [hunk 1]
```diff
@@ -65,14 +65,14 @@ <h4 class="title font-bold m">Easy Mart</h2>
         <main class="main grid-2-column flex-grow-1">
             <div class="flex flex-column flex-align-end pd-1">
                 <article class="cart-card m-2">
```
Comment (hsnice16) [inline Pages/cart page/cart.html]: Nice work on UI, very clean. Just one suggestion on Card, I think, you should not use the `article` tag for cards, as semantically it has a different meaning. `div` is fine for cards.

**Review by hsnice16 — APPROVED** [review]

**Review by Vishnu-Aithal — COMMENTED** [review]
Looks nice !
suggestion:
Some Padding inside the card for text would look better

File: Pages/cart page/cart.html (hunk 1 above)
Comment (BalajiHarinath) [reply Pages/cart page/cart.html]: Okay thanks for pointing this out, will change them to div.

**Comment by BalajiHarinath** [comment]
> Looks nice ! suggestion: Some Padding inside the card for text would look better

Okay will include some padding. Thanks for reviewing.

**Pull request merged**